src/components/SubmissionForm/ProjectForm.js

```diff
@@ -43,16 +42,9 @@ const ProjectForm = props => {
     e.preventDefault()
 
     let submitted_at = moment().format("MMMM Do YYYY, h:mm:ss a")
-    let display_image = state.display_image
-      ? state.display_image
-      : props.display_image
+    let display_image = props.display_image && props.display_image
```

Comment:
is this right?      props.display_image && props.display_image

Reply:
Yup! That's called a short circuit conditional, here's an article on it: https://codeburst.io/javascript-short-circuit-conditionals-bbc13ac3e9eb

Reply:
I was just about to ask about this.